src/material.cpp

```diff
@@ -282,6 +347,12 @@ void pyne::Material::write_hdf5(std::string filename, std::string datapath,
   // Read in nuclist if available, write it out if not
   //
   bool nucpath_exists = h5wrap::path_exists(db, nucpath);
+  // if no nucpath exists use the nested version...
+  if (!nucpath_exists) {
+    nucpath = datapath + "_" + nucpath.substr(1);  
```

Comment:
For a reason I ignore I am not able to use "/" to nest it properly....
(this is why I strip it from `nucpath` and add "_")

I am not satisfied by this, just submitting it to get advices....

Reply:
I am wondering if we want to add this to this PR, or if we shall issue an other one for it....

Reply:
if this works - let's stick with this but create an issue to introduce HDF5 groups for this in the future